s3/variables.tf

```diff
@@ -31,3 +31,11 @@ variable "tags" {
   type        = list(string)
   default     = []
 }
+
+variable "json_params" {
+  description = "A JSON string of arbitrary parameters"
+  type        = map(string)
```

Comment:
[question] is there a reason this one is `map(string)` and the other two are `map(object)`?

Reply:
Getting fixed in [this other PR](https://github.com/18F/terraform-cloudgov/pull/34)...